apache/superset
# fix(report-add): make to add error toast message when already attached report is added again into dashboard or chart

### SUMMARY
Attach new report to the dashboard again, but there are no error messages on the page #18225

### BEFORE/AFTER SCREENSHOTS OR ANIMATED GIF
BEFORE:
https://user-images.githubusercontent.com/47900232/158206908-7a8fe15e-31cf-4049-b7db-395d0755cecf.mov
![Screen Shot 2022-03-14 at 11 37 26 AM](https://user-images.githubusercontent.com/47900232/158207357-911d4574-ef99-4693-9e67-d0ede75dbf48.png)

AFTER:
![image](https://user-images.githubusercontent.com/47900232/157878392-297683c1-f19d-47ff-9aa6-6286c612c9c0.png)


### TESTING INSTRUCTIONS
**How to reproduce the bug**

1. Create a report for a chart or dashboard from the top right settings menu "Alerts & Reports" section - make sure you create a Report and not an alert
2. Go to the chart or dashboard that you created the report for
3. Click the calendar icon to schedule a report

### ADDITIONAL INFORMATION
